File: day12prob21.py
class Solution(object):
    def searchInsert(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: int
        """
        
        for i in range(len(nums)):
             
            # If K is found
            if nums[i] == target:
                return i
        # If arr[i] exceeds K
            elif nums[i] > target:
                return i
            
    
nums = [1,3,5,6]
target = 0
x = Solution()
print(x.searchInsert(nums, target))

File: day12prob21.java
public class day12prob21 {
    public int searchInsert(int[] nums, int target) {
        
        int k = nums.length;
        if(target > nums[k-1]){
            return nums.length;
        }
        for(int i = 0;i < nums.length; i++){
           if(nums[i] == target){
               return i;
           }
           else if(nums[i]>target){
               return i;
           }
        }
        return -1;
    }
}